static/js/Header.jsx

```diff
@@ -18,22 +18,74 @@ import {Autocomplete} from './Autocomplete'
 import { DropdownMenu, DropdownMenuSeparator, DropdownMenuItem, DropdownMenuItemWithIcon } from './common/DropdownMenu';
 
 
+const LoggedInDropdown = () => {
+
+  const getCurrentPage = () => {
+    return encodeURIComponent(Sefaria.util.currentPath());
+  }
+
+  return (
+    <DropdownMenu menuIconComponent={<ProfilePic
+                                          url={Sefaria.profile_pic_url}
+                                          name={Sefaria.full_name}
+                                          len={25}
+                              />
+    }>
+      <DropdownMenuItem>
+        <strong>{Sefaria.full_name}</strong>
+      </DropdownMenuItem>
+      <DropdownMenuSeparator />
+      <DropdownMenuItem url={'/settings/account'}>
+        <InterfaceText>Account Settings</InterfaceText>
+      </DropdownMenuItem>
+      <DropdownMenuItem url={'/torahtracker'}>
+        <InterfaceText text={{'en': 'Torah Tracker', 'he': 'לימוד במספרים'}} />
+      </DropdownMenuItem>
+      <DropdownMenuSeparator />
+      <div className="languageHeader">
+        <InterfaceText>Site Language</InterfaceText>
+      </div>
+      <div className='languageToggleFlexContainer'>
+        <DropdownMenuItem url={`/interface/english?next=${getCurrentPage()}`}>
+          English
+        </DropdownMenuItem>
+          <span className="languageDot">&#183;</span>
+        <DropdownMenuItem url={`/interface/hebrew?next=${getCurrentPage()}`}>
+        עברית    
+        </DropdownMenuItem>
+      </div>
+      <DropdownMenuSeparator />
+      <DropdownMenuItem url={'/notifications'}>
+        <InterfaceText text={{'en': 'New Additions', 'he': 'חידושים בארון הספרים של ספריא'}} />
+      </DropdownMenuItem>
+      <DropdownMenuItem url={'/help'}>
+        <InterfaceText text={{'en': 'Help', 'he': 'עזרה'}} />
+      </DropdownMenuItem>
+      <DropdownMenuSeparator/>
+      <DropdownMenuItem url={'/logout'}>
+        <InterfaceText text={{'en': 'Log Out', 'he': 'ניתוק'}} />
+      </DropdownMenuItem>
+    </DropdownMenu>
+  );
+}
+
+
 const ModuleSwitcher = () => {
   return (
-    <DropdownMenu>
-    <DropdownMenuItem url={'/'}>
+    <DropdownMenu menuIconComponent={<img src='/static/icons/module_switcher_icon.svg'/>}>
+    <DropdownMenuItem url={'/'} newTab={true}>
       <DropdownMenuItemWithIcon icon={'/static/icons/library_icon.svg'} textEn={'Library'} textHe={'ספריה'} />
     </DropdownMenuItem>
     <DropdownMenuSeparator />
-    <DropdownMenuItem url={'//sheets.sefaria.org'}>
+    <DropdownMenuItem url={'//sheets.sefaria.org'} newTab={true}>
```

Comment:
What are these double slashes?

Reply:
I don't recall, likely an error! Will remove.

Reply:
Might be `/sheets`, confirm with SK.